packages/widgetbook/lib/src/addons/device_addon/device_addon.dart

```diff
@@ -100,9 +100,3 @@ class DeviceAddon extends WidgetbookAddOn<DeviceSetting> {
     ).build(context, child);
   }
 }
-
-extension DeviceExtension on BuildContext {
-  Device? get device => getAddonValue<DeviceSetting>()!.activeDevice;
-
-  Orientation get orientation => getAddonValue<DeviceSetting>()!.orientation;
-}
```

Comment:
As you said, this will "lose" us access to the add-on. In what way could a developer have used this with the new implementation?

I'm not sad to see this code go. I think it was too complex, not scalable, and hard to understand. I'm just interested in the limitations the removal will inflict (if any)